standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2040,7 +2044,10 @@ struct CreateDatabaseRequest {
   9: optional i32 createTime,
   10: optional string managedLocationUri,
   11: optional string type,
-  12: optional string dataConnectorName
+  12: optional string dataConnectorName,
+  // use boolean skipFSWrites to decide whether create directory in file system or not
+  13: optional bool skipFSWrites=false,
+  14: optional Database database
```

Comment:
Do we need `optional Database database` here? or can we put the `skipFSWrites` into the struct `Database`, so that the old client can be able to skip create directory with less changes?

Reply:
Yes, we do. Because in the create database function with signature CreateDatabaseRequest, we will use getDatabase() function. This will need the `optional Database database`.